packages/fxa-content-server/tests/functional/settings_v2/external_links.js

```diff
@@ -5,20 +5,28 @@
 'use strict';
 
 const intern = require('intern').default;
+const config = intern._config;
+const SETTINGS_V2_URL = `${config.fxaContentRoot}beta/settings`;
```

Comment:
This already exists as `config.fxaSettingsV2Root`

Reply:
updated across the other instances in settingsv2 function test folder